src/index.js

```diff
@@ -0,0 +1,167 @@
+      const taskElement = document.getElementById(`${taskIndex}`);
+      const editTask = document.querySelector(`#edit${taskIndex}`);
+      editTask.classList.remove('hidden');
+      document.querySelector(`#task${taskIndex}`).classList.add('hidden');
+
+      taskElement.querySelector('.edit-manager').classList.remove('hidden');
+      taskElement.querySelector('.list-editor').classList.add('hidden');
+
+      editTask.addEventListener('keypress', (e) => {
+        const keypressed = (editTask) ? e.keyCode : e.which;
+        if (keypressed === 13) {
+          if (editTask.value !== '') { confirmEditTask(e); }
+        }
+      });
+    });
+  });
+
+  // Confirm task edition
+  document.querySelectorAll('.edit-confirm').forEach((ec) => {
+    ec.addEventListener('click', confirmEditTask);
+  });
+
+  // Cancel task etition
+  document.querySelectorAll('.edit-cancel').forEach((ec) => {
+    ec.addEventListener('click', (e) => {
+      const taskIndex = parseInt(e.target.parentNode.parentNode.id, 10);
+      const taskElement = document.getElementById(`${taskIndex}`);
+
+      document.querySelector(`#edit${taskIndex}`).classList.add('hidden');
+      document.querySelector(`#task${taskIndex}`).classList.remove('hidden');
+
+      taskElement.querySelector('.edit-manager').classList.add('hidden');
+      taskElement.querySelector('.list-editor').classList.remove('hidden');
+
+      printList();
+    });
+  });
+
+  // delete selected task
+  document.querySelectorAll('.delete-task').forEach((dtb) => {
+    dtb.addEventListener('click', (e) => {
+      const task = new Task();
+      const taskIndex = parseInt(e.target.parentNode.parentNode.id, 10);
+      task.Delete(taskIndex);
+      printList();
+    });
+  });
+
+  checkCompleted();
+  setCheckboxListener();
+};
+
+if (getToDoList() !== []) {
+  printList();
+}
+
+const addNewTask = () => {
+  if (newTaskInput.value !== '') {
+    toDoList = getToDoList();
+    const newIndex = toDoList.length + 1;
+    const task = new Task(newTaskInput.value, newIndex);
+    task.Add();
+    newTaskInput.value = '';
+    printList();
+  }
+};
+
+// Add new Task
+document.querySelector('#add').addEventListener('click', addNewTask);
+newTaskInput.addEventListener('keypress', (e) => {
+  const keypressed = (newTaskInput) ? e.keyCode : e.which;
+  if (keypressed === 13) {
+    if (newTaskInput.value !== '') { addNewTask(); }
+  }
+});
+
+// Clear all selected
+document.querySelector('#clear-list').addEventListener('click', () => {
+  let tasks = JSON.parse(localStorage.getItem('to_do_list'));
+  const checked = [];
+  tasks = tasks.filter((task) => {
+    if (task.completed === true) {
+      checked.push(task.index);
+      return false;
+    }
+    return true;
+  });
+  for (let i = checked.length - 1; i >= 0; i -= 1) {
+    tasks.forEach((task) => {
+      if (task.index > checked[i]) {
+        task.index -= 1;
+      }
+    });
+  }
```

Comment:
Avoid boring old for a loop. Example:
```
const list = [2, 5, 7, 2, 6, 2, 3, 5]
for (let i = 0; i < list.length; i++) {
    console.log(list[i])
}
```
Instead of the above code, we can use bellow:
```
list.forEach(number => console.log(number))
```